### PR TITLE
#22 - checkin/checkout function

### PR DESCRIPTION
**Pull request by OmarRoBra**
# Description
What did you implemented/Why did you implemented this:
 - Controllers and routes of logIn and logOut created.
 - Alerts and notice added to the login page.
 - Added functionality to the login page.

  
  ## Screenshots 
If applicable, include screenshots of the results or screenshots that help to see changes

image

image

image

### CODE REVIEW

**Review by martinmt-mx — DISMISSED** [review]
nice!

File: kata06/app/views/employees/login.html.erb [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
   </div>
 
 
+<% flash.each do |type, msg| %>
+  <div class="alert alert-info">
+    <%= msg %>
+  </div>
+<% end %>
 </div>
+<% end %>
```
Comment (YilKanda) [inline kata06/app/views/employees/login.html.erb]: Add a blank line.

File: kata06/app/controllers/employees_controller.rb [hunk 2]
```diff
@@ -41,4 +72,11 @@ def employee_params
...
     end
 
+    def login
+        @employee = Employee.new
+    end
+
+    
+
+
 end
```
Comment (YilKanda) [inline kata06/app/controllers/employees_controller.rb]: Add a blank line.

File: kata06/config/database.yml [hunk 3]
```diff
@@ -17,6 +17,9 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  host: localhost
+  username: postgres
+  password: alceconbotas2
```
Comment (YilKanda) [inline kata06/config/database.yml]: It is not recommended to leave this configuration in Git, what you can do is add this file (database.yml) and exclude it in Git through the .gitignore, and later add a database.yml.example file with the access configuration example to the database.

This is recommended, to the different configurations that each user has, on their computer, of PostgreSQL.

**Review by YilKanda — CHANGES_REQUESTED** [review]
Check the indentation of your *.html.erb and *.rb files (the latter with Rubocop), so that you can have the indentation of two spaces in your files.

File: kata06/config/database.yml (hunk 3 above)
Comment (OmarRoBra) [reply kata06/config/database.yml]: Ok, i will do the changes, thanks for the observations :D

**Review by yovanha — DISMISSED** [review]
Good job!

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by AlejandroVI99 — APPROVED** [review]
nice!

**Review by AlejandroVI99 — APPROVED** [review]

**Review by martinmt-mx — APPROVED** [review]

**Pull request merged**